.github/workflows/ooi_processing.yml

```diff
@@ -3,10 +3,30 @@ name: OOI processing
 on:
   # To run manually
   workflow_dispatch:
+    inputs:
+      node:
+        description: 'Alphanumeric node id (e.g. PC01A)'
+        required: false
+        default: 'PC01A'
+      start_time:
+        description: 'start_time formatted as Y-m-dTH-M-S'
+        required: false
+      end_time:
+        description: 'end_time formatted as Y-m-dTH-M-S'
+        required: false
+      segment_length:
+        description: 'Segment length in minutes'
+        required: false
+        default: 5
```

Comment:
Can time be smaller than a minute? Using fractions?  If we want spectrograms on a whale call length, it is usually only a few seconds?

Reply:
Yes, it is actually a float, so can be really small, I should probably indicate that in the description!